src/app/modules/consultations/consultation-profile/profile-card/profile-card.component.ts

```diff
@@ -171,16 +171,6 @@ export class ProfileCardComponent implements OnInit, OnChanges {
   }
 
   stepNext(hasResponseSubmited) {
-    if (!this.currentUser) {
-      this.cookieService.put('loginCallbackUrl', this.router.url === '/' ? '' : this.router.url);
```

Comment:
@preethijayanthi This cookie is actually important. Can you instead set it when a user visits a Consultation page?